static/src/javascripts/projects/common/modules/atoms/youtube-player.js

```diff
@@ -80,6 +84,27 @@ const onPlayerReadyEvent = (event, handlers: Handlers, el: ?HTMLElement) => {
     }
 };
 
+const createAdsConfig = (
+    adFree: boolean,
+    wantPersonalisedAds: boolean,
+    inPfpAdTargetingVariant: boolean
+): Object => {
```

Comment:
This returns ad-hoc an JavaScript object that meets YouTube's spec. It has to be typed thus to appease the Flow gods in this function's tests.